lib/qrda_qdm_template_validator.rb

```diff
@@ -4,207 +4,7 @@ module CqmValidators
   class QrdaQdmTemplateValidator
     include BaseValidator
 
-    # Hash of templateIds/extensions specified in the Patient Data Section QDM for QRDA R3
```

Comment:
Just checking that we don't anticipate QRDA Cat I <R5 being used anymore? Either for our projects, or any other users of this library?

Reply:
Doing this to be consistent with the libraries.  For older versions of QRDA, they would use an older version of this library.